pkg/client/client.go

```diff
@@ -16,14 +16,17 @@ import (
 	telemetrylib "github.com/SUSE/telemetry/pkg/lib"
 	"github.com/SUSE/telemetry/pkg/restapi"
 	"github.com/SUSE/telemetry/pkg/types"
+	"github.com/SUSE/telemetry/pkg/utils"
 )
 
 const (
 	//CONFIG_DIR  = "/etc/susetelemetry"
-	CONFIG_DIR      = "/tmp/susetelemetry"
-	CONFIG_PATH     = CONFIG_DIR + "/config.yaml"
-	AUTH_PATH       = CONFIG_DIR + "/auth.json"
-	INSTANCEID_PATH = CONFIG_DIR + "/instanceid"
+	CONFIG_DIR              = "/tmp/susetelemetry"
+	CONFIG_PATH             = CONFIG_DIR + "/config.yaml"
+	AUTH_PATH               = CONFIG_DIR + "/auth.json"
+	INSTANCEID_PATH         = CONFIG_DIR + "/instanceid"
+	TELEMETRY_DATA_MAX_SIZE = 10485760
```

Comment:
I would move these constant settings to pkg/lib/limits.go file, since they are related to the limits themselves.